ipfx/plot_qc_figures.py

```diff
@@ -145,7 +146,7 @@ def plot_single_ap_values(data_set, sweep_numbers, lims_features, sweep_features
             if nspikes:
                 plt.xlim(spikes[0]["threshold_t"] - 0.002, spikes[0]["fast_trough_t"] + 0.01)
         elif type_name == "short_square":
-            plt.xlim(stim_start - 0.002, stim_start + stim_dur + 0.01)
+            plt.xlim(stim_start_shifted - 0.002, stim_start_shifted + stim_dur + 0.01)
```

Comment:
What do the values `0.002` and `0.01` (same for the `1e-3` a few lines above) represent? Maybe consider making these `CONSTANT_VARIABLES`?

Reply:
I agree. Moreover, this entire module needs serious help. I will take care of it as a separate issue in the future.